<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/new/prefix1/win-xp/8.ico</normaloff>:/new/prefix1/win-xp/8.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/new/prefix1/win-xp/9.ico</normaloff>
      <normalon>:/new/prefix1/win-xp/7.ico</normalon>:/new/prefix1/win-xp/9.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>南京</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/new/prefix1/win-xp/52.ico</normalon>
     </iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ggg</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
